docs/CONTRIBUTING.md

```diff
@@ -66,6 +80,20 @@ After you are able to successfully build JsDbg, please follow the steps below to
 
 8. The JsDbg server will launch and prompt you to select a browser. If the debugger is attached to one browser, it is recommended that you choose a different browser for JsDbg.
 
+#### GDB
+
+After following the build instructions above, type `jsdbg` at the gdb prompt to run it. It will tell you the URL to open in your browser.
+
+Extensions should be automatically reloaded when you edit the source file; no need to restart the server. You can also reload the Python script without restarting if needed:
+```
+python
+import importlib
+importlib.reload(JsDbg)
+```
+
+For debugging, you may want to set verbose to true, either by editing `JsDbg.py` or running `python JsDbg.jsdbg.verbose = True` at the gdb prompt.
```

Comment:
By `debugging` here, are you referring to "using jsdbg to debug a program" or "debugging jsdbg itself"? I believe you are referring to the latter, but it'd be good to clear it up.

Reply:
Clarified that